package.json

```diff
@@ -7,6 +7,7 @@
     "test": "test"
   },
   "devDependencies": {
+    "core-assert": "^0.1.3",
```

Comment:
Is it? Without that `deepStrictEqual` is not defined because zuul depends on the assert package which is not up to date with node's assert (https://nodejs.org/api/assert.html).

```
`-- zuul@3.10.1
  `-- browserify@13.0.0
    `-- assert@1.3.0
```

It seems like the assert package is yours as well though so maybe you can tell me more about why it's missing deepStrictEqual :)

And BTW in case it's not clear, I need deepStrictEqual because otherwise an object with the same shape as an array is considered equal to the array, which is not quite valid.